scala-xml/src/test/scala/scalaxml/ScalaXmlSpec.scala

```diff
@@ -29,6 +31,14 @@ class ScalaXmlSpec extends Http4sSpec {
       getBody(resp.body) must_== ("html".getBytes)
     }
 
+    "parse XML in parallel" in {
+      // https://github.com/http4s/http4s/issues/1209
+      val pool = Executors.newFixedThreadPool(5)
```

Comment:
[org.http4s.Http4sSpec.TestPool](https://github.com/http4s/http4s/blob/release-0.16.x/testing/src/test/scala/org/http4s/Http4sSpec.scala#L145) is the implementation I believe.